internal/provider/schema_cloudflare_access_application.go

```diff
@@ -37,15 +37,17 @@ func resourceCloudflareAccessApplicationSchema() map[string]*schema.Schema {
 			Description: "Friendly name of the Access Application.",
 		},
 		"domain": {
-			Type:        schema.TypeString,
-			Required:    true,
+			Type:     schema.TypeString,
+			Optional: true,
+			Computed: true, // REVIEW
+			// Required:    true,
```

Comment:
```suggestion
			Computed: true,
```

Reply:
We still need to be able to set it for `self_hosted`, wouldn't that break it?

Reply:
that would; if we need to set it, this is fine.